de.zwarf.picplanner.json

```diff
@@ -0,0 +1,123 @@
+{
+    "app-id" : "de.zwarf.picplanner",
```

Comment:
Does that app-id mean the requirements? I get "Diese Domain wurde geparkt." on `zwarf.de`

Reply:
I own this domain. The website for PicPlanner will be online soon.

Reply:
I placed a "place holder" website until the normal website is done.